addon/utils/url-builder.js

```diff
@@ -1,8 +1,22 @@
-import param from 'jquery-param';
+import URL from 'url';
```

Comment:
Could add a note about this being a polyfill and that when Ember drop support for IE 11 this won't be needed anymore:

https://caniuse.com/#search=URLSearchParams